shared/login/register/code-page/index.desktop.js

```diff
@@ -112,6 +113,7 @@ const CodePageEnterText = ({
       multiline={true}
       value={enterText}
       onChangeText={onChangeText}
+      onKeyDown={onKeyDown}
```

Comment:
I added this typing `onKeyDown: (e: SyntheticKeyboardEvent<>, enterText: string) => void`, but Flow gave me the following error for this line:

```
Cannot create Input element because string [1] is incompatible with boolean [2] in the second argument of property onKeyDown.
```

Any suggestions?